src/components/nav/menuListItems.jsx

```diff
@@ -11,12 +11,11 @@ function MenuListItems() {
   };
 
   return (
-    <ul className="nav-list">
+    <ul className={style.navUl}>
       {Menu.map((menuItem, index) => (
-        <li className="nav--list" key={menuItem.id}>
+        <li className={style.navLi} key={menuItem.id}>

```

Comment:
`nav` si `item` sunt niste nume mai potrivite decat `navUl` si `navLi` . incearca sa denumesti clasa de CSS in functie de rol, nu elementu HMLT folosit